device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/InternalClient.java

```diff
@@ -45,7 +50,7 @@
         "Direct methods are only supported over MQTT, MQTT_WS, AMQPS, and AMQPS_WS";
 
     DeviceClientConfig config;
-    DeviceIO deviceIO;
+    private DeviceIO deviceIO;
```

Comment:
DeviceClient and ModuleClient now cannot access this field directly, which will help protect us from making any silly mistakes later when adding device/module APIs